docs/migration-guide.rst

```diff
@@ -139,6 +139,9 @@ Other backwards incompatible changes affecting plugins
         def load(self, info):
             ModelImporter.registerModel('job', Job, 'jobs')
 
+* In the web client, ``girder.rest.restRequest`` no longer accepts the deprecated ``path``
```

Comment:
We should put this under a new heading for web client changes

Reply:
@zachmullen I agree, but can we do this in another PR for reorganizing the migration guide? Right now, this change belongs under "Other backwards incompatible changes affecting plugins" as much as changes to mail template detection, etc.